test/upgrade/main.go

```diff
@@ -277,7 +303,14 @@ func runConfigWalker(ctx context.Context, validConfigs []*configTest) {
 			log.Fatalf("PollUntilContextTimeout timed out while attempting upgrade to Agones version %s. Helm Status %s",
 				config.agonesVersion, helmStatus)
 		}
+
+		go createGameServers(cancelCtx, config.gameServerPath)
 	}
+	// Allow some soak time at the last version
```

Comment:
For the versions that are not the last one, should we also wait for a while after a version is installed and then upgrade to the next version?

Reply:
We certainly could do. It takes ~1 min to install a new version, so the Game Servers are installing on the existing version for a certain percentage of that time until the controllers roll over to the new version, but we can increase that time.